Baconit/Strings/id-ID/Resources.resw

```diff
@@ -0,0 +1,889 @@
+    <value>New</value>
+  </data>
+  <data name="CMOld.Text" xml:space="preserve">
+    <value>Old</value>
+  </data>
+  <data name="CMQA.Text" xml:space="preserve">
+    <value>Tanya Jawab</value>
+  </data>
+  <data name="CMTop.Text" xml:space="preserve">
+    <value>Terbaik</value>
+  </data>
+  <data name="collapseTBN.Text" xml:space="preserve">
+    <value>sembunyikan</value>
+  </data>
+  <data name="CopyPerFMI.Text" xml:space="preserve">
+    <value>Salin permalink</value>
+  </data>
+  <data name="CountTB.Text" xml:space="preserve">
+    <value>Count:</value>
+  </data>
+  <data name="CPermaFMI.Text" xml:space="preserve">
+    <value>Salin permalink</value>
+  </data>
+  <data name="DeletePostClick.Text" xml:space="preserve">
+    <value>Hapus posting</value>
+  </data>
+  <data name="EditPostClick.Text" xml:space="preserve">
+    <value>Ralat posting</value>
+  </data>
+  <data name="MoreTB.Text" xml:space="preserve">
+    <value>lebih</value>
+  </data>
+  <data name="RunGilded.Text" xml:space="preserve">
+    <value>diemaskan</value>
+  </data>
+  <data name="RunSticky.Text" xml:space="preserve">
+    <value>tempelan</value>
+  </data>
+  <data name="ShareCommentFMI.Text" xml:space="preserve">
+    <value>Bagikan komentar</value>
+  </data>
+  <data name="SharePostFMI.Text" xml:space="preserve">
+    <value>Bagikan posting</value>
+  </data>
+  <data name="TapViewEntire.Text" xml:space="preserve">
+    <value>Tekan di sini untuk melihat semuanya</value>
+  </data>
+  <data name="AboutDescription.Text" xml:space="preserve">
+    <value>Reddunt adalah klien Reddit® yang indah berdasarkan Baconit, sebuah aplikasi bersumber terbuka. Tersedia di semua perangkat Windows 10.
+Fokus utama berada di optimasi dan fitur-fitur khusus Windows 10!</value>
+  </data>
+  <data name="AddMessagesTB.Text" xml:space="preserve">
+    <value>Tambahkan pesan ke pusat pemberitahuan secara diam-diam</value>
+  </data>
+  <data name="AlwaysCombo.Content" xml:space="preserve">
+    <value>Selalu</value>
+  </data>
+  <data name="AuthorFilter.PlaceholderText" xml:space="preserve">
+    <value>Filter Penulis</value>
+  </data>
+  <data name="AutoTB.Text" xml:space="preserve">
+    <value>Perkecil header secara otomatis saat di komentar</value>
+  </data>
+  <data name="AutoUpdateTB.Text" xml:space="preserve">
+    <value>Ubah gambar kunci layar secara otomatis</value>
+  </data>
+  <data name="AutoUpdateWTB.Text" xml:space="preserve">
+    <value>Ubah latar belakang secara otomatis</value>
+  </data>
+  <data name="BackgroundInboxCTB.Text" xml:space="preserve">
+    <value>Cek kotak masuk secara berkala</value>
+  </data>
+  <data name="BackgroundInboxTB.Text" xml:space="preserve">
+    <value>Cek kotak masuk secara berkala</value>
+  </data>
+  <data name="CAllTimeB.Content" xml:space="preserve">
+    <value>Sepanjang Waktu</value>
+  </data>
+  <data name="CancelCT.Content" xml:space="preserve">
+    <value>BAtal</value>
```

Comment:
`BAtal` is correct ?

Reply:
Ah, sorry, that one's a typo. It's supposed to be `Batal`.